rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml

```diff
@@ -0,0 +1,53 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verify services are up and checking in
+- name: Verify all nova services are up
+  shell: |
+    . ~/openrc
+    openstack compute service list | grep -vE '\+|State' | awk -F '|' '{print $7}' | tr -d '[[:blank:]]'
```

Comment:
Yep, something like:

```
openstack compute service list -f csv -c "Id" -c "Binary" -c "Host" -c "State"
```

I think it'd be good to include more than just the `State`, so that if a service is down you can see it in the output rather than having to now go and dig into it.